invenio_db/utils.py

```diff
@@ -78,3 +81,11 @@ def create_alembic_version_table():
         alembic.migration_context._ensure_version_table()
         for head in alembic.script_directory.revision_map._real_heads:
             alembic.migration_context.stamp(alembic.script_directory, head)
+
+
+def drop_alembic_version_table():
```

Comment:
Wouldn't it be useful if we also had a combined function ``drop_all()`` that does ``db.drop_all()`` and ``drop_alembic_version_table()`` so that we can use a single function from tests?